packages/@aws-cdk/aws-lambda/lib/code.ts

```diff
@@ -10,60 +10,103 @@ export abstract class Code {
    * @param key The object key
    * @param objectVersion Optional S3 object version
    */
-  public static bucket(bucket: s3.IBucket, key: string, objectVersion?: string): S3Code {
+  public static fromBucket(bucket: s3.IBucket, key: string, objectVersion?: string): S3Code {
     return new S3Code(bucket, key, objectVersion);
   }
 
+  // /**
+  //  * @deprecated use `fromBucket`
+  //  */
+  // public static bucket(bucket: s3.IBucket, key: string, objectVersion?: string): S3Code {
+  //   return this.fromBucket(bucket, key, objectVersion);
+  // }
```

Comment:
Gotta uncomment those -- otherwise that's full of breaking changes on an APIs that I reckon is marked `stable`.